packages/commons/src/auth/authenticationMiddleware.ts

```diff
@@ -27,12 +27,14 @@ export const authenticationMiddleware: (
 
     try {
       const jwtToken = jwtFromAuthHeader(req, ctx.logger);
-      const valid = await verifyJwtToken(jwtToken, config, ctx.logger);
-      if (!valid) {
-        throw unauthorizedError("Invalid token");
+      const { decoded } = await verifyJwtToken(jwtToken, config, ctx.logger);
+
+      if (!decoded) {
+        throw unauthorizedError(`Invalid token: ${JSON.stringify(decoded)}`);
```

Comment:
Just make sure it's ok to print the entire decoded token, we may want to print only some relevant claims